### DISCUSSION TITLE
[Coreshop] The best way to add additional information to shipment

### DISCUSSION THREAD

**Discussion by twin-elements** [pimcore/pimcore#17070] [Q&A / Support]
I need to add additional information to the shipment data and show it in the admin panel.
There is no delivery address, there is only a pickup number (pickup id), which I have to add to the data for delivery

Could someone give me an example?
Thanks.

**Comment by dpfaffenbauer** [comment]
What exactly do you want to add? Shipment is just a Pimcore DataObject and you can add fields in the backend. You can also extend the extjs backend form if you want that

**Reply by twin-elements** [reply]
When the user selects a carrier at checkout, he must enter the name (ID) of the pickup (the point to which the package will be delivered).
Is it best to create a data brick and add that data to the additional order data?
If so, how do you display the pickup id in the panel next to the delivery and invoice address?

**Reply by dpfaffenbauer** [reply]
Can you contact me directly to figure out what you need and how we can solve that? We can post the solution here then later for others to see
